docs/tutorials/getting_started.rst

```diff
@@ -86,601 +117,85 @@ instance, for Mark 5B files one needs the number of channels used, as well as
 
-    >>> fhns = vdif.open(SAMPLE_VDIF, 'rs', squeeze=False)
-    >>> fhns.sample_shape    # Sample shape now keeps channel dimension.
-    SampleShape(nthread=8, nchan=1)
-    >>> fhns.ndim            # fh.shape and fh.ndim also change with squeezing.
-    3
-    >>> d2 = fhns.read(12)
-    >>> d2.shape             # Decoded data has channel dimension.
-    (12, 8, 1)
-    >>> fhns.close()
+    >>> fh_vdif.seek(100)
+    100
 
-Basic information about the file is obtained by either by ``fh.info`` or simply
-``fh`` itself::
+Seeking from the end or current position is also possible, using the same
+syntax as for typical file objects. It is also possible to seek in units of
+time::
 
-    >>> fh.info
-    Stream information:
-    start_time = 2014-06-16T05:56:07.000000000
-    stop_time = 2014-06-16T05:56:07.001250000
-    sample_rate = 32.0 MHz
-    shape = (40000, 8)
-    format = vdif
-    bps = 2
-    complex_data = False
-    <BLANKLINE>
-    File information:
-    edv = 3
-    frame_rate = 1600.0 Hz
-    samples_per_frame = 20000
-    sample_shape = (8, 1)
-
-    >>> fh
-    <VDIFStreamReader name=... offset=12
-        sample_rate=32.0 MHz, samples_per_frame=20000,
-        sample_shape=SampleShape(nthread=8),
-        bps=2, complex_data=False, edv=3, station=65532,
-        start_time=2014-06-16T05:56:07.000000000>
-
-Not coincidentally, the first is identical to what we :ref:`found above
-<getting_started_inspecting>` using `~baseband.file_info`.
-
-The filehandle itself also shows the ``offset``, the current location of the
-sample file pointer. Above, it is at ``12`` since we have read in 12 (complete)
-samples.  If we called ``fh.read (12)`` again we would get the next 12 samples.
-If we instead called ``fh.read()``, it would read from the pointer's *current*
-position to the end of the file.  If we wanted all the data in one array, we
-would move the file pointer back to the start of file, using ``fh.seek``,
-before reading::
-
-    >>> fh.seek(0)      # Seek to sample 0.  Seek returns its offset in counts.
-    0
-    >>> d_complete = fh.read()
-    >>> d_complete.shape
-    (40000, 8)
+    >>> fh_vdif.seek(-1000, 2)    # Seek 1000 samples from end.
+    39000
+    >>> fh_vdif.seek(10*u.us, 1)    # Seek 10 us from current position.
+    39320
 
-We can also move the pointer with respect to the end of file by passing ``2``
-as a second argument::
-
-    >>> fh.seek(-100, 2)    # Second arg is 0 (start of file) by default.
-    39900
-    >>> d_end = fh.read(100)
-    >>> np.array_equal(d_complete[-100:], d_end)
-    True
-
-``-100`` means 100 samples before the end of file, so ``d_end`` is equal to
-the last 100 entries of ``d_complete``.  Baseband only keeps the most recently
-accessed data frame in memory, making it possible to analyze (normally large)
-files through selective decoding using ``seek`` and ``read``.
-
-.. note::
+``fh_vdif.tell`` returns the current offset in samples or in time::
```

Comment:
Given that you just showed seeking in units of `us`, maybe show the same for `tell`?

Reply:
Implemented.